papergres.go

```diff
@@ -688,6 +705,7 @@ func exec(q *Query, nonQuery bool) *Result {
 			if err != nil {
 				return err
 			}
+			meta.LastInsertId, _ = res.LastInsertId()
```

Comment:
What result is going to be unaccounted for here? Is it an error? Why are we not properly handling it?

Reply:
Good catch!
The error that is not being captured here from the `LastInsertID()` method could only arise if the database driver does not support it.  All relational sql database drivers support it by default like postgres, mysql,  sqlite etc. And this method in turn depends on the Go's official database/sql package and the `lib/pq` package. 

Just for curiosity, here's how an error would look like if the driver does not implement it.
```
return 0, errors.New("LastInsertId is not supported by this driver")
```

But since all database drivers implement it,  it's safe to not catch that error. And even if we don't,  the default value of `0` zero returned by the package suffices and which in turn should be interpreted by client programs.

Reply:
It's best to send the error that was generated by the package.

Reply:
I've had problems trying to catch errors that should have been generated by Papergres in that pass. This was one of the reasons why I couldn't catch it. The error that should have been returned was not being captured and things fell through the crack.

Reply:
Cool. I sent a new fix with the changes to capture these errors as well. Please do review one more time.